core/camel-api/src/main/java/org/apache/camel/TimeoutMap.java

```diff
@@ -19,7 +19,7 @@
 /**
  * Represents a map of values which timeout after a period of inactivity.
  */
-public interface TimeoutMap<K, V> extends Runnable {
+public interface TimeoutMap<K, V> extends Service {
```

Comment:
Didn't really want to add `Service` here but it's required to decorate `DefaultTimeoutMap`